golang/x/relay/keeper/querier.go

```diff
@@ -34,8 +60,14 @@ func queryIsAncestor(ctx sdk.Context, path []string, req abci.RequestQuery, keep
 
 	// getAncestor is removed by the handler switch/case block above
 
+	// check that the path is this many items long, error if not
+	if len(path) > 3 {
+		return []byte{}, types.ErrTooManyArguments(types.DefaultCodespace)
+	} else if len(path) < 3 {
```

Comment:
if the path is 2 items long, accept it, but don't call `strconv.Atoi(path[2])` later. instead, use `15` as a default

Reply:
Fixed